GameEngine/src/main/java/dk/sdu/se/f23/InVasion/main/Game.java

```diff
@@ -52,7 +52,7 @@ public void create() {
             plugin.onEnable(gameData, world);
             System.out.println(plugin.getClass().getName() + " loaded");
         }
-        gameData.setPlayerMoney(5000);
+        gameData.setPlayerMoney(200);
```

Comment:
Should be 0, but okay...

Reply:
disagree
200 > 0
NO more should be said